composer.json

```diff
@@ -43,6 +43,11 @@
             "Laravel\\Octane\\Tests\\": "tests"
         }
     },
+    "scripts": {
+        "post-autoload-dump": [
+            "@php vendor/bin/testbench package:discover --ansi"
```

Comment:
This won't work and will fail on production as testbench isn't available. I think we need to do some things from the docs below? 

https://packages.tools/testbench/basic/testcase.html#package-auto-discovery

Reply:
This script won't be used on production though, will it? 🤔 And in production, people would be using a path to their app, so it wouldn't be an issue? Or am I missing something?

Reply:
Right, my bad!